nautobot_golden_config/jobs.py

```diff
@@ -150,7 +150,16 @@ def run(self, data, commit):
 
         # Commit / Push each repo after job is completed.
         for intended_repo in intended_repos:
-            self.log_debug(f"Push new intended configs to repo {intended_repo.url}.")
+            self.logger.info(
+                "%s: repo updated",
+                intended_repo.obj.name,
+                extra={
+                    "grouping": "GC Repo Commit and Push",
+                    "object": intended_repo.obj,
+                },
+            )
+
+            self.log_debug(f"Push new intended configs to repo {intended_repo['repo_obj'].nautobot_repo_obj.name}.")
```

Comment:
Need to update the `log_debug` calls to remove the fstring as well. Also, please test both jobs and post screenshots for both.